README.md

```diff
@@ -1 +1,82 @@
+  * `commons-to-core-mapping/MappingCommonsToIOF` **[Released]**
+    > This ontology maps the OMG Common Ontologies concepts to those of IOF Core.
+  * `commons-to-core-mapping/meta/MappingAnnotationVocabularyToCommons.rdf` **[Released]**
+    > The IOF Mapping Annotation Vocabulary to Commons maps the AV to the Object Management Group (OMG)&apos;s Annotation Vocabulary provided in the Commons Ontology Library. The Commons Annotation Vocabulary is a subset of what is included in the IOF AV, but is used across a number of OMG and other emerging standards and thus the mapping makes it easier to use other Commons library ontologies in an IOF context.
+* `maintenance/Maintenance` **[Provisional]**
+  > The purpose of this IOF Maintenance Reference ontology is to support semantic interoperability through the use of modular ontologies in the maintenance domain. This Ontology contains terms identified as common in a number of application ontologies for maintenance management, maintenance procedures, asset failure, and failure modes and effects analysis. The ontology is based on the IOF Core Ontology.
+* `supplychain/SupplyChainReferenceOntology` **[Provisional]**
+  > Supply Chain Reference Ontology (SCRO) aims to extend the IOF Core with the constructs (classes and properties) related to the domain of supply chain and logistics. The purpose of the ontology is to serve as a foundation for ensuring consistency and interoperability across various supply chain and logistics application ontologies.
+## Reasoner
+
+IOF ontologies, provisional and released, have been verified to be logically consistent and satisfiable with Hermit version 1.4.3 reasoner, plugin to Protege 5.5
+
+## In Development
+
+* `productionplanning/mfg-planning`
+* `productservicesystems/pss-ontology2`
+
+# Installing / Getting started
+
+The minimal setup you need, short of reading raw XML, is a suitable ontology viewer or editing tool, installed on your client machine. IOF recommends a desktop version of the open-source tool [Protégé](https://protege.stanford.edu/), but other open source and commercial tools are known to work as well. The repository for this ontology includes the necessary files for opening the ontology in Protégé without warnings and errors. The same is not guaranteed for other tools.  
+
+# Getting Involved
+
+## General Discussions
+
+The IOF welcomes those organizations and persons who would like to contribute to this and other IOF ontology projects. To start contributing, or to join the general discussions, please see [Getting Involved](https://oagiscore.atlassian.net/wiki/spaces/IOF/overview). 
+
+Getting involved will also give you access to discussions on issues regarding issues not raised through GitHub, and the status of any issues posted in these release notes at time of release.
```

Comment:
This sentence should either be dropped or if you keep it, it should read more like: "Getting involved will give you access to the discussions the various IOF working groups have related to issues raised here in the public GitHub, and their general efforts to evolve the IOF ontologies for future releases."